Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -1037,38 +1037,64 @@ void Detector::addAlignableVolumesLayer(int lr, TString& parent, Int_t& lastUID)
   // Add alignable volumes for a Layer and its daughters
   //
   // Created:      06 Mar 2018  Mario Sitta First version (mainly ported from AliRoot)
+  // Updated:      06 Jul 2021  Mario Sitta Do not set Layer as alignable volume
   //
 
   TString wrpV =
     mWrapperLayerId[lr] != -1 ? Form("%s%d_1", GeometryTGeo::getITSWrapVolPattern(), mWrapperLayerId[lr]) : "";
   TString path = Form("%s/%s/%s%d_1", parent.Data(), wrpV.Data(), GeometryTGeo::getITSLayerPattern(), lr);
   TString sname = GeometryTGeo::composeSymNameLayer(lr);
```

Comment:
This line should not be needed anymore

Reply:
Hi @shahor02 it was just a reminder that the layer was alignable and now it isn't anymore. But ok, I can remove it. Do you agree to do it in the next PR ? Otherwise I trigger again all the tests... :)
Many thanks. Cheers